### PR TITLE
ROLLUP-158: Add support for RDOC 

### PR DESCRIPTION
**Pull request by AndresQuijano**
## What

-   Hardcode the value of RDOC in the price feeder to always return 1.

## Why

-  Add support for RDOC as token.

## Refs

-   [ROLLUP-158](https://rsklabs.atlassian.net/browse/ROLLUP-158)

### CODE REVIEW

File: core/bin/zksync_api/src/fee_ticker/tests.rs [hunk 1]
```diff
@@ -581,7 +582,7 @@ async fn test_error_coingecko_api() {
         Default::default(),
         FakeTokenWatcher,
     );
-    let connection_pool = ConnectionPool::new(Some(1));
+    let connection_pool: ConnectionPool = ConnectionPool::new(Some(1));
```
Comment (franciscotobar) [inline core/bin/zksync_api/src/fee_ticker/tests.rs:585]:
```suggestion
    let connection_pool = ConnectionPool::new(Some(1));
```
Should we declare type or let Rust infer automatically.

File: core/bin/zksync_api/src/fee_ticker/tests.rs [hunk 2]
```diff
@@ -677,3 +678,49 @@ async fn test_error_api() {
...
+    const BASE_URL: &str = "http://127.0.0.1:9876";
+    const RDOC_SYMBOL: &str = "RDOC";
+
+    let client = reqwest::ClientBuilder::new()
+        .timeout(CONNECTION_TIMEOUT)
+        .connect_timeout(CONNECTION_TIMEOUT)
+        .build()
+        .expect("Failed to build reqwest::Client");
+
+    env::set_var("DATABASE_URL", DATABASE_URL);
```
Comment (franciscotobar) [inline core/bin/zksync_api/src/fee_ticker/tests.rs:695]: Im not sure about this, I got the same error but just when tried to run it directly, but by running it with the zk command, it works properly.

File: core/bin/zksync_api/src/fee_ticker/tests.rs [hunk 3]
```diff
@@ -677,3 +678,49 @@ async fn test_error_api() {
         .await
         .unwrap();
 }
+
+#[tokio::test]
+#[ignore]
+async fn test_rdoc_price() {
+    const DATABASE_URL: &str = "postgres://postgres@localhost/plasma";
+    const BASE_URL: &str = "http://127.0.0.1:9876";
```
Comment (franciscotobar) [inline core/bin/zksync_api/src/fee_ticker/tests.rs:686]: What is the BASE_URL?

File: core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs [hunk 4]
```diff
@@ -316,3 +315,91 @@ impl<T: TokenPriceAPI + Send + Sync> FeeTickerAPI for TickerApi<T> {
...
+    use std::env;
+    use zksync_types::{Address, Token, TokenId, TokenKind, TokenPrice};
+
+    #[tokio::test]
+    async fn should_return_one_for_rdoc() {
+        const DATABASE_URL: &str = "postgres://postgres@localhost/plasma";
+        const RDOC_SYMBOL: &str = "RDOC";
+        const RDOC_VALUE: u32 = 1;
+
+        env::set_var("DATABASE_URL", DATABASE_URL);
```
Comment (franciscotobar) [inline core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs:332]: Im not sure about this, I got the same error but just when tried to run it directly, but by running it with the zk command, it works properly.

File: core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs [hunk 5]
```diff
@@ -316,3 +315,91 @@ impl<T: TokenPriceAPI + Send + Sync> FeeTickerAPI for TickerApi<T> {
...
+        assert_eq!(actual_qoute.usd_price.to_u32().unwrap(), RDOC_VALUE);
+    }
+
+    #[tokio::test]
+    async fn should_return_value_from_cache() {
+        const DATABASE_URL: &str = "postgres://postgres@localhost/plasma";
+        const TEST_TOKEN_SYMBOL: &str = "TEST";
+        const TOKEN_VALUE: u32 = 5;
+
+        env::set_var("DATABASE_URL", DATABASE_URL);
```
Comment (franciscotobar) [inline core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs:361]: Im not sure about this, I got the same error but just when tried to run it directly, but by running it with the zk command, it works properly.

File: core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs [hunk 6]
```diff
@@ -316,3 +315,91 @@ impl<T: TokenPriceAPI + Send + Sync> FeeTickerAPI for TickerApi<T> {
...
+        let connection_pool = ConnectionPool::new(Some(1));
+        let ticker_api = TickerApi::new(connection_pool, FakeTickerApi);
+
+        let price_cache_map = HashMap::from([(
+            test_token.id,
+            TokenCacheEntry::new(test_token_price.clone(), Instant::now()),
+        )]);
+        let price_cache = Arc::new(Mutex::new(price_cache_map));
+
+        let ticker_api = TickerApi::with_price_cache(ticker_api, price_cache);
```
Comment (franciscotobar) [inline core/bin/zksync_api/src/fee_ticker/ticker_api/mod.rs:397]: nice :)

File: core/bin/zksync_api/src/fee_ticker/tests.rs (hunk 1 above)
Comment (AndresQuijano) [reply core/bin/zksync_api/src/fee_ticker/tests.rs:585]: Changed.

**Pull request closed without merging**